src/middleware/validation.middleware.ts

```diff
@@ -48,7 +48,7 @@ export const checkValidations = async (req: Request, res: Response, next: NextFu
         [ResignedOnDateKey]: prepareData(req.body, ResignedOnDateKeys),
       };
 
-      const routePath = req.route.path;
+      const routePath = req.route.path.includes(config.DYNAMIC_SUB_PATH) ? req.route.path.replace(config.DYNAMIC_SUB_PATH, "") : req.route.path;
```

Comment:
there is also a req.route.path in the checkTrustValidations function, does that need to be updated as well?

Reply:
This was a problem here because the trustee pages have part of the url dynamic, I don't think this is the case for the url using checkTrustValidations